README.md

```diff
@@ -51,14 +51,24 @@ make plugins.json
 
 In addition to running as a standalone server, the Marketplace is also designed to run as a Lambda function, compiling the `plugins.json` database into the binary for immediate access without further configuration.
 
+### Automatic Deployment
+
+Changes merged to `master` are automatically deployed to https://api.staging.integrations.mattermost.com.
+
+Changes merged to `production` are automatically deployed to https://api.integrations.mattermost.com.
+
+When adding or updating the plugins database (or corresponding tooling), submit the changes directly to `production`, and then merge `production` immediately back to `master` to reduce unnecessary merge conflicts. All other changes should be committed directly to `master`. Changes pending release from `master` should be merged to `production` after qualification and in coordination with any supporting Mattermost server release.
```

Comment:
Is there a difference between merging `production` to `master` and cherry-picking the lastest commit from `production` into `master`?

Reply:
The only difference in my mind is that the former catches anything that might have been missed, forcing an earlier reconciliation. If we wanted to go down the cherry-pick route, I'd suggest we adopt the mattermost-server semantics, commit everything to `master`, and cherry-pick to production instead. Thoughts?

(I'll merge this regardless, and we can iterate as needed.)